drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -321,6 +321,26 @@ function DragAndDropTemplates(configuration) {
         );
     };
 
+    var assessmentNotificationTemplate = function (ctx) {
+        if (configuration.mode === DragAndDropBlock.STANDARD_MODE) {
+            return
+        }
+        var icon = h('div.icon' , h('i.fa.fa-exclamation'));
+        var button = h('div.action', h('button.close-assessment-notification', gettext("Continue")));
+         var attemptText = h('p',
+            gettext(
+                "You can review answers / resubmit {max_attempts} times"
+            ).replace("{max_attempts}", ctx.max_attempts)
```

Comment:
If `ctx.max_attempts` is null, this shows up as "You can review answers / resbumit null times.". I think the language should be changes in case of null or 0 `max_attempts` to be:
"You can review answers / resubmit as many times as you want."